src/vulkan-renderer/application.cpp

```diff
@@ -414,11 +414,11 @@ VkResult Application::init() {
 
             // We have to explicitly load this function.
             PFN_vkCreateDebugReportCallbackEXT vkCreateDebugReportCallbackEXT =
-                reinterpret_cast<PFN_vkCreateDebugReportCallbackEXT>(vkGetInstanceProcAddr(instance, "vkCreateDebugReportCallbackEXT"));
+                reinterpret_cast<PFN_vkCreateDebugReportCallbackEXT>(vkGetInstanceProcAddr(vkinstance->get_instance(), "vkCreateDebugReportCallbackEXT"));
 
             if (nullptr != vkCreateDebugReportCallbackEXT) {
```

Comment:
Should be `if (vkCreateDebugReportCallbackEXT != nullptr)`. Or alternatively, just `if (vkCreateDebugReportCallbackEXT)`.

Reply:
I will use the second choice.

Reply:
done.